CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 Contributions are **welcome** and will be fully **credited**.
 
-We accept contributions via Pull Requests on [Github](https://github.com/GA/BrokerAPI).
+We accept contributions via Pull Requests on [Github](https://github.com/pivotal-cf/brokerapi).
```

Comment:
the URL is this one: https://github.com/asyncapi/php-template make sure to updated it please :)